Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -561,6 +561,21 @@ ApplePayEnabled:
   sharedPreferenceForWebProcess: true
   richJavaScript: true
 
+AriaIDLNewReflection:
```

Comment:
ARIA should be capitalized. I don't think "IDL" is necessary but we should be explicitly about the fac this isa bout enumerated attribute so `EnumeratedARIAAttributeReflection` would be a better name.